kohese-portal/client/cache-worker/cache-worker.ts

```diff
@@ -958,7 +958,6 @@ async function fetchMissingCacheInformation(missingCacheData){
       const timestamp = response.timestamp;
       timestamp.responseReceiptTime = responseReceiptTime;
       console.log('::: Response for getMissingCacheInfo');
-      // tslint:disable-next-line:forin
```

Comment:
Why are all of the tslint lines removed?

Reply:
tslint is fully deprecated in angular 13. angular.json no longer recognizes "lint" as a valid configuration option. I haven't checked for implementation, but it favors esLint now.